todo/src/AddTodoForm.js

```diff
@@ -14,8 +15,7 @@ function AddTodoForm({onAddTodo}) {
   };
   return (
     <form onSubmit={handleAddTodo}>
-        <label htmlFor="todoTitle">Title</label>
-        <input id="todoTitle" type="text" name="title" value={todoTitle} onChange={handleTitleChange}/>
+        <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange} >Title</InputWithLabel>
```

Comment:
Looking good! 👍

You're only missing the id, value, and isFocused prop here. It'd be something like this:
`<InputWithLabel id={"whateverYouWant"} value={todoTitle} handleTitleChange={handleTitleChange} isFocused-{true}>Title</InputWithLabel>`
The `id` prop will allow two instances of the InputWithLabel to be created. Also, passing in isFocused prop to so the component knows if it should autoFocus. And lastly, you can pass the `todoTitle` as a `value` prop. 
That's a mouth full! Sorry for the long comment. 😄 I hope this helps